### PR TITLE
suggestions to the landing page

### PR DESCRIPTION
**Pull request by stevemar**
* make the agenda a table
* add credits
* add tested platforms

### CODE REVIEW

**Comment by stevemar** [comment]
this has been rebased

**Comment by stevemar** [comment]
@scottdangelo poke

File: workshop/README.md [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
-  * Visualize with charts and graphs
-* **Machine Learning**
-  * Build a model with Spark
-  * Deploying a model with Watson Machine Learning
-  * Testing the model with built-in UI
-  * (Optional) Create a Python Flask app that uses the model
-* **Monitoring models**
-  * Monitor model with Watson OpenScale
+|   |   |
+| - | - |
```
Comment (scottdangelo) [inline workshop/README.md]: Needs the required `Pre-work` section.

Comment (stevemar) [reply workshop/README.md]: done

**Pull request merged**